### DISCUSSION TITLE
After cloning the repo, a few files and folders are not present, which is causing some issues.

### DISCUSSION THREAD

**Discussion by SamiranDas2004** [realworld-apps/realworld#1592] [Community]
// import {LoginPayload} from "~/models/login-payload.model";
// import { RegisteredUser } from '../app/models/registered-user.model';
//import { createUser } from '../app/services/auth.service';
//import { addComment, createArticle } from '../app/services/article.service';

After cloning the repo, I noticed that some files mentioned in import statements are missing. I'd like to contribute to this project and need some initial guidance on how to proceed with the missing files.

![Screenshot 2024-09-28 065920](https://github.com/user-attachments/assets/53657cdc-142c-4b29-995b-463e54cd68d7)
![Screenshot 2024-09-28 065855](https://github.com/user-attachments/assets/f5f617c2-427d-4913-b661-4ad78a36a80b)

**Comment by geromegrignon** [comment]
Hello, I'm migrating the API to Nitro currently, explaining these issues.
